app/views/_includes/summary-cards/industry-expertise.html

```diff
@@ -16,27 +16,12 @@
 {{ xGovukSummaryCard({
   
   rows: [
-    {
-      key: {
-        text: "Do you have any industry or occupational expertise?"
-      },
-      value: { 
-        text: data.anyIndustryExpertise
-      },
-      actions: {
-        items: [{
-          text: "Change",
-          visuallyHiddenText: "do you have any industry or occupational expertise?",
-          href: "/application/industry-expertise"
-        }]
-      }
-    },
     {
       key: {
         text: "Provide details of your industry or occupational expertise"
       },
       value: { 
-        text: data.industryExpertiseDetails
+        text: data.anyIndustryExpertise
```

Comment:
this should be 'data.industryExpertiseDetails' as it needs to pull the data from the industry-expertise > add-details page.

Reply:
Thanks, I have corrected it.